src/app/donation-start/donation-start.component.html

```diff
@@ -264,8 +264,9 @@ <h1 class="b-rh-2 b-bold">{{ campaign.title }}</h1>
 
               <mat-form-field>
                 <mat-label for="billingPostcode">Billing postcode</mat-label>
-                <input formControlName="billingPostcode" id="billingPostcode" matInput>
+                <input formControlName="billingPostcode" id="billingPostcode" matInput (change)="onBillingPostCodeChanged($event)">
               </mat-form-field>
+              <div class="error" *ngIf="paymentGroup.get('billingPostcode')?.errors?.invalidBillingPostCode">Post code doesn't match the one used for your card.</div>
```

Comment:
I still don't think we need a distinct message here. The Stripe error copy is already surfaced and has been used with donors without issue, whereas I think this wording and having 2x errors shown could both be confusing.